app/controllers/vm_common.rb

```diff
@@ -1677,6 +1677,39 @@ def set_form_vars
     session[:edit] = @edit
   end
 
+  ACTION_TYPES = {
+    "advanced_settings"   => ->(amount) { n_("Advanced Setting", "Advanced Settings", amount) },
```

Comment:
The very nature of this change is troubling to me. We're adding a big string map *to the code*, and in doing so throwing away some rather useful-looking context. I find it implausible that we can reliably translate "Group" without ambiguity.

I guess I'd like to understand why we don't just move the map straight into gettext, even for English, and use `n_('ui_action|groups')`?

I appreciate that we want to kill off the default i18n yaml file, because it conflicts with our translation pipeline. But I'm not convinced dumping huge string maps into our controllers is the way to do that: it seems to solve one problem by creating another.